src/v/kafka/server/tests/fetch_plan_bench.cc

```diff
@@ -272,10 +267,3 @@ PERF_TEST_F(fetch_plan, t1p100_no_auth) { return run_bench(1, 100, false); }
 PERF_TEST_F(fetch_plan, t1p100_yes_auth) { return run_bench(1, 100, true); }
 PERF_TEST_F(fetch_plan, t100p1_no_auth) { return run_bench(100, 1, false); }
 PERF_TEST_F(fetch_plan, t100p1_yes_auth) { return run_bench(100, 1, true); }
-
-PERF_TEST_F(fetch_plan, max_no_auth) {
```

Comment:
I removed this test as I think it's pretty pointless and not clear what it does from the name. It never really produced any different results than the t1p100 one.